src/components/Icon/Icon.js

```diff
@@ -8,26 +9,44 @@ import './Icon.css';
  * @uxpinwrappers
  * SkipContainerWrapper
  */
-const Icon = ({click, mouseDown, mouseUp, mouseLeave, mouseOver, icon, color, size, uxpinRef}) => (
-  <chi-icon
-    icon={icon}
-    color={color}
-    size={size}
-    onClick={click}
-    onMouseEnter={mouseOver}
-    onMouseLeave={mouseLeave}
-    onMouseDown={mouseDown}
-    onMouseUp={mouseUp}>
+const Icon = ({tooltipMessage, tooltipPosition, click, mouseDown, mouseUp, mouseLeave, mouseOver, icon, color, size, uxpinRef}) => {
+  const uuid = uuid4();
+
+  if (tooltipMessage && tooltipPosition !== 'default') {
+    const initialize = setInterval(() => {
+      if (window.chi && document.getElementById(uuid)) {
+        window.chi.tooltip(document.getElementById(uuid));
+        clearInterval(initialize);
+      }
+    }, 1000);
+  }
+
+  return (
+    <chi-icon
+      id={uuid}
+      icon={icon}
+      color={color}
+      size={size}
+      onClick={click}
+      data-tooltip={tooltipMessage}
+      data-position={tooltipPosition}
+      onMouseEnter={mouseOver}
+      onMouseLeave={mouseLeave}
+      onMouseDown={mouseDown}
+      onMouseUp={mouseUp}>
       <span className="-sr--only">
         i
       </span>
     </chi-icon>
-);
+  )
+};
 
 Icon.propTypes = {
   size: PropTypes.oneOf(['xs', 'sm', 'sm--2', 'sm--3', 'md', 'lg', 'xl', 'xxl']),
   icon: PropTypes.string,
   color: PropTypes.oneOf(['primary', 'secondary', 'dark', 'light', 'info', 'grey', 'muted', 'success', 'warning', 'danger', 'navy', 'orange']),
+  tooltipMessage: PropTypes.string,
+  tooltipPosition: PropTypes.oneOf(['default', 'top', 'right', 'bottom', 'left']),
```

Comment:
```suggestion
  tooltipPosition: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
```

Reply:
@josecarlosgonzalezv As per the discussion with Tom, he did not wanted tooltip text to be shown on top position by default, it must be empty. Only upon selecting a particular tooltip position from dropdown should reveal the tooltip on the icon component. Hence, i have implemented the default property.

Reply:
@josecarlosgonzalezv As you mentioned, had a discussion with Tom today, changing the default position to top to be consistent with other components.